scripts/newinstall.sh

```diff
@@ -187,11 +187,12 @@ fi
 
 if true; then
 	# Check the version by running a small Python program (taken from the Python EUPS package)
+    # XXX should not assume that python is in the default path
```

Comment:
OK. Like the check in the EUPS package. Please tweak the comment to something like: "XXX this will break if python is not in $PATH". Also, indenting is wrong (the file is full of tabs).